src/server.c

```diff
@@ -5125,6 +5197,39 @@ sds genRedisInfoString(const char *section) {
         raxStop(&ri);
     }
 
+    if ((allsections || !strcasecmp(section,"latencystats"))) {
+        /* Latency by percentile distribution per command category */
+        if (sections++) info = sdscat(info,"\r\n");
+        info = sdscatprintf(info, "# Latencystats - latency by percentile distribution\r\n");
+        struct redisCommand *c;
+        dictEntry *de;
+        dictIterator *di;
+        di = dictGetSafeIterator(server.commands);
+        while((de = dictNext(di)) != NULL) {
+            char *tmpsafe;
+            c = (struct redisCommand *) dictGetVal(de);
+            if (!c->latency_histogram)
+                continue;
+            info = fillPercentileDistributionLatencies(info,getSafeInfoString(c->name, strlen(c->name), &tmpsafe),c->latency_histogram);
+            if (tmpsafe != NULL) zfree(tmpsafe);
+        }
+        dictReleaseIterator(di);
+
+        /* Per command category cumulative distribution of latencies */
+        if (sections++) info = sdscat(info,"\r\n");
+        info = sdscatprintf(info, "# Latencystats - cumulative distribution of latencies\r\n");
```

Comment:
this is a new concept (one section filter name, producing two sections.
is there any reason not to include all the info in one section? (which has identical, one word name, which is exactly the same as it's filter string)

Reply:
> is there any reason not to include all the info in one section?

No. addressed it in the latest commit